### DISCUSSION TITLE
Feature request: Allow setting "Internal" for nupkg packages and downloading inside an org with GITHUB_TOKEN

### DISCUSSION THREAD

**Discussion by sheershoff** [community/community#20382] [Actions]
I see that for organizations there is a setting to allow package repos to be "Internal" with a hope that GITHUB_TOKEN in GitHub Actions would allow downloading any package from a channel with "Internal" type from an org's repo.

However, docs only cover docker images repos for this. We, specifically, need it for nupkg channels. There's no such setting in nupkg packages/repos/channels.

This would alleviate the need for a robot user with an issued PAT.

**Comment by kylerjensen** [comment]
My organization also needs this for Maven and NPM packages. We're using a "bot" user with a PAT in our organization, but it makes us frustrated that we have to pay for an additional seat just for this purpose.
